show/main.py

```diff
@@ -737,8 +744,29 @@ def mac(vlan, port, verbose):
     if port is not None:
         cmd += " -p {}".format(port)
 
+    if address is not None:
+        cmd += " -a {}".format(address)
+
+    if type is not None:
+        cmd += " -t {}".format(type)
+
+    if count:
+        cmd += " -c"
+
     run_command(cmd, display_cmd=verbose)
 
+@mac.command('aging-time')
+@click.pass_context
+def aging_time(ctx):
+    app_db = SonicV2Connector()
+    app_db.connect(app_db.APPL_DB)
+    table = "SWITCH_TABLE*"
+    keys = app_db.keys(app_db.APPL_DB, table)
+
+    for key in keys:
+        fdb_aging_time = app_db.get(app_db.APPL_DB, key, 'fdb_aging_time')
+        if fdb_aging_time is not None:
+            click.echo("Aging time for {} is {} seconds".format(key.split(':')[-1], fdb_aging_time))
```

Comment:
I would suggest giving some explicit hints in case no ageing time is configured.

Reply:
Done